Makefile

```diff
@@ -72,6 +72,17 @@ MOCKERY=mockery
 
 .DEFAULT_GOAL := test-and-lint
 
+BASE_IMAGE_MULTIARCH := localhost:5000/baseimg:$(VERSION)-$(shell echo $(ROOT_IMAGE) | tr : -)
+PLATFORMS=linux/amd64,linux/s390x
+INGESTER_TAG=$(subst JAGERCOMP,ingester,$(IMAGE_TAGS))
```

Comment:
Yes, I think we need to, since each jaeger component has its unique name, hence the unique tag value.

`JAGERCOMP` is not a variable but a const string. We use it in the tag computation procedure in `scripts/build-upload-docker-images.sh` to get the value of variable `IMAGE_TAGS`. Then in `Makefile`  `L77-L84`, we replace it with the name of each jaeger component in variable `IMAGE_TAGS` to get each actual tag value.

Reply:
I am not a fan of these cross-dependences. If the string that contains `JAGERCOMP` is defined in the script, why can't the script resolve it as needed, without pushing the substitution into Makefile?

Also, Jaeger is spelled with 'e'.

Reply:
Got it. I've corrected the spelling and moved the related code to the script.